client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -171,16 +175,17 @@ const EmailProvidersStackedComparison = ( {
 				onIntervalChange={ changeIntervalLength }
 			/>
 
-			{ hasExistingEmailForwards && domainsWithForwards !== undefined && (
+			{ ! hasCartDomain && hasExistingEmailForwards && domainsWithForwards !== undefined && (
```

Comment:
We don't need to test `hasCartDomain` since we've already done that for `hasExistingEmailForwards` above.